spring-kafka/src/main/java/org/springframework/kafka/listener/KafkaMessageListenerContainer.java

```diff
@@ -674,6 +674,18 @@ else if (listener instanceof MessageListener) {
 			this.subBatchPerPartition = setupSubBatchPerPartition();
 		}
 
+		private Properties propertiesFromProperties() {
+			Properties props = new Properties(this.containerProperties.getKafkaConsumerProperties());
+			Set<String> stringPropertyNames = props.stringPropertyNames();
+			// Copy non-string-valued properties from the default hash table to the new properties object
+			this.containerProperties.getKafkaConsumerProperties().forEach((key, value) -> {
```

Comment:
Let's extract local variable for this `this.containerProperties.getKafkaConsumerProperties()` !